tor/helpers/flair.py

```diff
@@ -142,6 +142,8 @@ def set_meta_flair_on_other_posts(config):
             )
             flair_post(post, flair.meta)
             send_to_slack(
-                f'New meta post: <{post.url}|{post.fullname}>',
+                f':thinking: '
+                f'New meta post: <{post.url}|{post.submission.shortlink}> '
```

Comment:
I get that `post.fullname` just outputs the internal PRAW id, but what about the post title instead? We're already making this clickable as a link, wouldn't it be more helpful to annotate the link with the title of the post?

Reply:
I originally meant to put the title when I wrote this, but got those mixed up somehow. +1

Reply:
Ah, crap. I totally forgot to change that -- I'll change it to title instead.